lint-staged.config.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { execSync } from 'child_process';
+
+export default {
+    // Runs ESLint with fix on all staged JS/TS files
+    '*.{js,ts}': (files) => {
```

Comment:
So lint-staged is great because it actually shows you the files it's running against so you don't need to echo the file list. It also enables parallelization so unless the command requires it, you don't have to pass the files in manually.

For example:

```
"*.js": ["eslint", "prettier --write"]
```